server/stats.go

```diff
@@ -131,6 +131,56 @@ func (s *Server) GetStatValue(ctx context.Context, in *pb.GetStatValueRequest) (
 	return &pb.GetStatValueResponse{Value: result}, nil
 }
 
+// GetStatSeries implements API for Mixer.GetStatSeries.
+func (s *Server) GetStatSeries(ctx context.Context, in *pb.GetStatSeriesRequest) (
```

Comment:
I feel we should also consider deduping these 3 (and more coming?) "mostly same functions. Add a TODO?

Reply:
done